suzieq/poller/controller/manager/static.py

```diff
@@ -41,6 +42,25 @@ def __init__(self, config_data: Dict = None, validate: bool = True):
 
         self._workers_count = config_data.get("workers", 1)
 
+        # We need a pipeline thats at least as big as the number of workers
+        max_cmd_pipeline = config_data['config-dict'] \
+            .get('poller', {}) \
+            .get('max-cmd-pipeline', 0)
+        if ((max_cmd_pipeline != 0) and
+                (max_cmd_pipeline % self._workers_count != 0)):
+            logger.error(f'max-cmd-pipeline({max_cmd_pipeline}) has to be a '
+                         'multiple of the number of workers'
+                         f'({self._workers_count})')
+            sys.exit(1)
```

Comment:
Please raise an exception instead:
```suggestion
           raise SqPollerConfError(f'max-cmd-pipeline({max_cmd_pipeline}) has to be a '
                         'multiple of the number of workers'
                         f'({self._workers_count})')
```